### PR TITLE
Sabrina branch

### PR DESCRIPTION
**Pull request by brucker3**

### CODE REVIEW

**Review by zpatel-dev — CHANGES_REQUESTED** [review]
How can I exclude myenv directory from approval or merging with the master? Or do Sabrina have to do another pull request?

**Comment by brucker3** [comment]
> How can I exclude myenv directory from approval or merging with the master? Or do Sabrina have to do another pull request?

is myenv your python virtual enviroment?
I added mine to the gitignore file. you can find it in the proejct rout directory is ls -a

File: checkers/manage.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```
Comment (brucker3) [inline checkers/manage.py:1]: this file is not needed it exist at the root layer mysite

File: checkers/checkers/settings.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
+"""
```
Comment (brucker3) [inline checkers/checkers/settings.py:1]: this file looks good but I'm confused why its in another checkers folder?

File: checkers/baseapp/models.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+from django.db import models
```
Comment (brucker3) [inline checkers/baseapp/models.py:1]: what is the directory baseapp/? it looks like this code should be in mysite/checkers/modelrs.py

**Comment by brucker3** [comment]
this pr is no longer needed

**Pull request closed without merging**